src/coreComponents/constitutive/fluid/multifluid/compositional/functions/CompositionalProperties.hpp

```diff
@@ -125,6 +121,9 @@ struct CompositionalProperties
                                   real64 & dMassDensity_dp,
                                   real64 & dMassDensity_dt,
                                   arraySlice1d< real64 > const & dMassDensity_dz );
+
+private:
+  static constexpr real64 gasConstant = MultiFluidConstants::gasConstant;
```

Comment:
As it allows refactoring and you've to pay the price of `#include` anyway, why not bypassing unscoped `gasConstant` and replace its occurences by `MultiFluidConstants::gasConstant`. That would be easier to maintain.